instrumentation/google.golang.org/grpc/otelgrpc/grpctrace.go

```diff
@@ -43,6 +46,10 @@ type config struct {
 	Filter         Filter
 	Propagators    propagation.TextMapPropagator
 	TracerProvider trace.TracerProvider
+	MeterProvider  metric.MeterProvider
+
+	meter             metric.Meter
+	rpcServerDuration syncfloat64.Histogram
```

Comment:
This config struct isn't only for tracing anymore. How about moving it to a `config.go` file, along which the config bits that are generic?

Reply:
That makes sense.

Reply:
I rename the `grpctrace.go` to `config.go`, and also move the content in `grpcmetric.go` into `config.go`

@dmathieu